django_react_proj/users-fe/src/components/UserList/UserList.js

```diff
@@ -1,70 +1,32 @@
 import React from "react";
 import { Table } from "reactstrap";
-import axios from "axios";
-import { API_URL } from "../../constants";
-import NewUserModal from "../NewUserModal/NewUserModal";
-import ConfirmRemovalModal from "../ConfirmRemovalModal/ConfirmRemovalModal";
-import TaskButton from "../TaskButtonModal/TaskButtonModal";
-import styles from "./UserList.module.css"
+import UsersListRow from "../UsersListRow/UsersListRow";
 
-const UserList = ({
-  users = [],
-  resetState
-}) => {
-  const deleteUser = async(id) => {
-    try{
-      await axios.delete(`${API_URL}${id}/`)
-      resetState()
-    } catch (error){
-      console.log(error)
-    }
-  };
-
-return (
-  <Table dark>
-    <thead>
-      <tr>
-        <th>Nazwa zadania</th>
-        <th>Opis</th>
-        <th>Data rejestracji</th>
-        <th></th>
-      </tr>
-    </thead>
-    <tbody>
-      {users.length === 0 ? (
+const UserList = ({ users = [], resetState }) => {
```

Comment:
This can be `=> (<Table dark> [...]</Table>)`. For arrow function if you put and expression right after the arrow a return keyword can be omitted.